README.md

```diff
@@ -181,56 +181,54 @@ Note: this is a strawman meant to stimulate discussion. Feel free to knock it do
 
 The *master* branch must always be in a production-ready state. Features landed must be complete and functional.
 
-Every Release Branch creates an opportunity to establish a new Long Term Support release. For instance, suppose the master branch rolls to version `2.0.0`. The LTS WG can decide in advance that there will be an LTS release in the `2.0.x` stream. At any point, the LTS WG can declare any version within that stream to be an "LTS Candidate" by cutting a `2.0.x-rc.1` tag where `x` is the current patch level. Sufficient time should then be given to allow the tag to be reviewed and tested. If things look good, the `2.0.x` version becomes the LTS Release. If, after testing, it becomes apparent that additional patches or even minor bumps become necessary, the LTS Candidate can be shifted and new `2.y.x-rc.2` tag created where `y` is the current minor and `x` is the current patch. LTS Releases are identified by appending a `~0.0.0` to the version (i.e. `2.5.3~0.0.0`). This additional metadata identifies the post-release semantic versioning extension for the LTS release.
+Release Branches occur at every Major.Minor version bump (e.g. `2.0`, `2.1`, `2.2`, etc). Every Release Branch creates an opportunity to establish a new Long Term Support release.
 
-Once the `-rc.1` tag is cut at a particular point, the master stream goes into "Feature Freeze" until the release branch is cut. New features landed before the `-rc.1` tag make it into the LTS Release, those that do not must wait to land until after the Feature Freeze is lifted.
+For instance, suppose the master branch rolls to version `2.0.0`. The LTS WG can decide in advance that there will be an LTS release in the `2.0.x` stream. At any point, the LTS WG can declare any version within that stream to be an "LTS Candidate" by cutting a `2.0-rc.1` tag. Sufficient time should then be given to allow the tag to be reviewed and tested. If things look good, the tagged version becomes the LTS Release. If, after testing, it becomes apparent that additional patches become necessary, the LTS Candidate can be shifted and a new `2.0-rc.2` at a specific patch level.
```

Comment:
> At any point, the LTS WG can declare any version within that stream to be an "LTS Candidate" by cutting a `2.0-rc.1` tag.

It would be better to just choose it when a new minor rolls out. Far less confusing, makes less tension with master, etc.